frontend/packages/dev-console/src/components/monitoring/alerts/MonitoringAlerts.scss

```diff
@@ -7,6 +9,14 @@
   &--kebab {
     text-align: right;
   }
+  .odc-monitoring-alerts {
+    &--notification {
+      width: 20%;
+      @media screen and (max-width: $pf-global--breakpoint--2xl) {
+        width: 30%;
+      }
+    }
+  }
```

Comment:
```suggestion
    &--notification {
      width: 20%;
      @media screen and (max-width: $pf-global--breakpoint--2xl) {
        width: 30%;
      }
    }
```

Reply:
it is intentional so that I can define a property to nested class. Changed the class name so that we have no confusion.